ibis/expr/api.py

```diff
@@ -1622,12 +1622,14 @@ def difference(table: ir.Table, *rest: ir.Table, distinct: bool = True):
 Use this object to refer to a previous table expression in a chain of
 expressions.
 
-!!! note "`_` may conflict with other idioms in Python"
+::: {.callout-note}
+## `_` may conflict with other idioms in Python
```

Comment:
nit: adding ". You can consider using `ibis._` directly instead" or similar?

Reply:
Done.